src/server.ts

```diff
@@ -10,33 +10,61 @@ import type { GraphQLContext } from './context';
 
 let LOG_LEVEL = (process.env.LOG_LEVEL as LogLevel) || 'info';
 
-export function buildServer(context: GraphQLContext) {
+function initJaegerProvider() {
+  let provider = undefined;
+  if (process.env.ENABLE_JAEGER) {
+    provider = buildProvider();
+    if (!process.env.JAEGER_ENDPOINT) {
+      throw new Error(
+        'Jaeger endpoint not found. Please ensure that the Jaeger endpoint is properly configured and available.'
+      );
+    }
+    if (!process.env.JAEGER_SERVICE_NAME) {
+      throw new Error(
+        'Jaeger service name not found. Please ensure that the Jaeger service name is properly configured and available.'
+      );
+    }
+  }
+  return provider;
+}
+
+function buildPlugins() {
   let plugins = [];
+
   plugins.push(useGraphQlJit());
-  plugins.push(
-    useOpenTelemetry(
-      {
-        resolvers: false, // Tracks resolvers calls, and tracks resolvers thrown errors
-        variables: true, // Includes the operation variables values as part of the metadata collected
-        result: true, // Includes execution result object as part of the metadata collected
-      },
-      process.env.ENABLE_JAEGER ? buildProvider() : undefined
-    )
-  );
-  if (process.env.ENABLE_INTROSPECTION !== 'true')
+
+  if (process.env.ENABLE_LOGGING === 'true') {
```

Comment:
Should I see any errors logged in stdout/stderr in case of GQL issues after query processing and in case if logging is enabled?
Example response:

```graphql
{
	"errors": [
		{
			"message": "Cannot query field \"data1\" on type \"EventData\". Did you mean \"data\"?",
			"locations": [
				{
					"line": 21,
					"column": 7
				}
			]
		}
	]
}
```

Because I don't see anything in terminal's console.

Reply:
Good call, fixed!

Reply:
What about logging in stdout comms like before:

```shell
Server is running on port: 8080
{
  traceId: 'c3ec72623751bcf03e4b8b95be6fc76e',
  parentId: undefined,
  name: 'Anonymous Operation',
  id: '7e46dab48c942a7c',
  kind: 1,
  timestamp: 1676815549542771,
  duration: 31731,
  attributes: {
    'graphql.execute.document': 'query getEvents {\n' +
      '  events(\n' +
      '    input: {address: "B62qrZRZY1GuqR6gfGEN1reEvCTi1MXJ6HNUkHg9XohWMWUNJzwksdd", to: 1000, from: 0, status: ALL}\n' +
      '  ) {\n' +
      '    blockInfo {\n' +
      '      height\n' +
      '      stateHash\n' +
      '      parentHash\n' +
      '      ledgerHash\n' +
      '      chainStatus\n' +
      '      timestamp\n' +
      '      globalSlotSinceGenesis\n' +
      '      globalSlotSinceHardfork\n' +
      '    }\n' +
      '    transactionInfo {\n' +
      '      hash\n' +
      '      memo\n' +
      '      status\n' +
      '      authorizationKind\n' +
      '    }\n' +
      '    eventData {\n' +
      '      index\n' +
      '    }\n' +
      '  }\n' +
      '}',
    'graphql.execute.variables': '{}'
  },
  status: { code: 0 },
  events: [ { name: 'exception', attributes: [Object], time: [Array] } ]
}
ERR Error: connect ECONNREFUSED 192.168.31.69:5432
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1187:16)
    at cachedError (/Users/shymkiv/projects/o1/Archive-Node-API/node_modules/postgres/cjs/src/query.js:171:23)
    at new Query (/Users/shymkiv/projects/o1/Archive-Node-API/node_modules/postgres/cjs/src/query.js:36:24)
    at sql (/Users/shymkiv/projects/o1/Archive-Node-API/node_modules/postgres/cjs/src/index.js:111:11)
    at getEventsQuery (/Users/shymkiv/projects/o1/Archive-Node-API/build/src/db/archive-node-adapter/queries.js:107:22) {
  locations: [ { line: 2, column: 3 } ],
  path: [ 'events' ]
}
```
Because logging enabled for Yoga client doesn't actually logs comms as before, only:

```shell
DEBUG Parsing request to extract GraphQL parameters                                                                                                        │
DEBUG Processing GraphQL Parameters                                                                                                                        │
DEBUG Processing GraphQL Parameters done.                                                                                                                  │
DEBUG Parsing request to extract GraphQL parameters                                                                                                        │
DEBUG Processing GraphQL Parameters                                                                                                                        │
DEBUG Processing GraphQL Parameters done.
```

No matter if issue happened with request or not.

Reply:
Gotcha! The current behaviour is if `Jaeger` is enabled, all logs will be forwarded to the server. If `Jaeger` is not enabled, stdout will be used instead. But now, errors are logged to the server as well if they occur, whether Jaeger is configured or not.